M2/Macaulay2/m2/examples.m2

```diff
@@ -169,11 +170,35 @@ extractExamples = docBody -> (
 -- examples: get a list of examples in a documentation node
 -----------------------------------------------------------------------------
 
+-- low-level unexported method called by getExampleOutput
+examples' := method(Dispatch => Thing)
+examples' Hypertext   := stack @@ extractExamplesLoop
+examples' String      := examples' @@ makeDocumentTag
+examples' DocumentTag := tag  -> (
+    rawdoc := fetchAnyRawDocumentation tag;
+    ?? examples' DIV {rawdoc.Description})
```

Comment:
I think I get what is happening here, but it's a bit too terse and bad for readability. Maybe just write `try ...`?